EIPS/eip-3076.md

```diff
@@ -0,0 +1,292 @@
+---
+eip: 3076
+title: Validator client interchange format (slashing protection)
+author: Michael Sproul (@michaelsproul), Sacha Saint-Leger (@sachayves), Danny Ryan (@djrtwo)
+discussions-to: https://discord.com/channels/595666850260713488/598292067260825641/
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-10-27
+---
+
+## Simple Summary
+
+A JSON interchange format for Serenity (eth2) that contains the necessary slashing protection information required to safely migrate keys between clients.
+
+## Abstract
+
+ A standard format for transferring a key's signing history allows validators to easily switch between clients without the risk of signing conflicting messages. While a [common keystore format](https://eips.ethereum.org/EIPS/eip-2335) provides part of the solution, it does not contain any information about a key's signing history. For a validator moving their keys from client A to client B, this can lead to scenarios in which client B inadvertently signs a message that conflicts with an earlier message signed with client A.  
```

Comment:
Keep in mind that linking to a draft EIP creates a weak dependency here that may cause problems if you want to move this to final before 2335 becomes final.  If the only reason for mentioning it is as comparison, I recommend just leaving it out as it isn't really meaningful information for developers 5-10 years in the future who are reading this EIP (target audience).